DashWallet/Sources/Models/CrowdNode/TxFilters/CoinsToAddressTxFilter.swift

```diff
@@ -17,7 +17,7 @@
 
 public class CoinsToAddressTxFilter: TransactionFilter {
```

Comment:
Just for the future, when you know that you won't extend some class. It better to mark it as final, in this case swift in most cases will do a direct call to some functions, otherwise it will use dynamic dispatching which is slower that direct calls. Here is more info: https://github.com/apple/swift/blob/main/docs/OptimizationTips.rst#reducing-dynamic-dispatch